waterloop-site/src/components/SustainableTech/NavBar/SideBar.tsx

```diff
@@ -145,55 +141,73 @@ class Sidebar extends React.Component<MyProps> {
         </StyledToggle>
         <ListContainer className={this.props.sidebarOpen ? '' : 'closed'}>
           <div>
-            <IconBlack src={Waterlooplogo}></IconBlack>
-            <Link
-              to="/"
+            <ScrollLink
               onClick={(): void => {
                 this.props.handleClickSidebar(false);
+                setTimeout(function () {
```

Comment:
Let's use arrow functions instead of named functions (all other instances as well).